include/miopen/miopen.h

```diff
@@ -5663,24 +5663,52 @@ MIOPEN_EXPORT miopenStatus_t miopenSumForward(miopenHandle_t handle,
 
 #ifdef MIOPEN_BETA_API
 
-/*! @ingroup argmax
- * @brief Find the index of the maximum value of a tensor across dimensions.
+/*! @ingroup ReduceExtreme
+ * @enum miopenReduceExtremeOp_t
+ * Reduction Extreme operation types
+ */
+typedef enum
+{
+    MIOPEN_REDUCE_EXTREME_ARGMIN =
+        1, /*!< the operation is getting the minimum index of the reduced elements */
+    MIOPEN_REDUCE_EXTREME_ARGMAX =
+        2, /*!< the operation is getting the maximum index of the reduced elements */
+    MIOPEN_REDUCE_EXTREME_MIN =
+        3, /*!< the operation is getting the minimum value and index of the reduced elements */
+    MIOPEN_REDUCE_EXTREME_MAX =
+        4, /*!< the operation is getting the maximum value and index of the reduced elements */
+} miopenReduceExtremeOp_t;
+
+// ReduceExtreme APIs
+/** @addtogroup ReduceExtreme
+ *
+ *  @{
+ */
+
+/*! @brief Find the the extreme(minimum, maximum) value and index of a tensor across dimensions.
  *
  * @param handle                   MIOpen handle (input)
  * @param xDesc                    Tensor descriptor for data input tensor x (input)
  * @param x                        Data tensor x (input)
  * @param dim                      Dimensions to reduce argmax. (input)
```

Comment:
"Dimensions" is incorrect as dim may hold only one value, so this comment must be fixed. Does dim contain the index of the dimension to be reduced? Or we reduce the range of dimensions [0,dim]?

Please also check other places where dim is documented.

Reply:
[resolved]